### ISSUE TITLE
Release 2.1.11 did not include fix for react-native

### ISSUE THREAD

**Issue by eugenemlee** [ai/nanoid#185]
Using nanoid on RN throws an error saying react native does not have a built-in secure random generator. This is with `import 'react-native-get-random-values';` included as well.

![image](https://user-images.githubusercontent.com/4890571/77160572-939e4e00-6ae2-11ea-8c09-f0bded7edfbc.png)

Notice that the error message is different from `index.browser.js` in 9b4453180d178a21a9e677765949d39e293b6048 that was supposed to fix the issue. I had a look at the nanoid package in my `node_modules` and `index.browser.js` is missing code from the commit.

```javascript
if (
    typeof navigator !== 'undefined' &&
    navigator.product === 'ReactNative' &&
    !self.crypto  // <--- this is missing
  ) {
...
```
Perhaps this needs to be re-released on npm?

Thanks!

**Comment by ai** [comment]
Yeap, we are making big refactoring right now.

I will release this fix this week. I will ping you to test a new release.

**Comment by ai** [comment]
Can you help me with React Native and my new TS/ESM release system in Nano Events?
https://github.com/ai/nanoevents/issues/44

I will release new Nano ID, which I will polish my TS/ESM system on other projects.

**Comment by ai** [comment]
The fix was released in Nano ID 3.0
